api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -360,6 +360,21 @@ message UIIntegrationEnrollCompleteEvent {
   IntegrationEnrollMetadata metadata = 1;
 }
 
+// EditorChangeEvent is an event that is emitted when a user role set changes resulting in
+// a editor role being added on removed
+message EditorChangeEvent {
```

Comment:
Wrong proto? This should be in prehog/v1alpha/teleport.proto, right?

Reply:
Yes, thanks! I forgot to commit `prehog/v1alpha/teleport.proto`, thank you.

Adding the event to `usageevents.proto` is not necessary since this event is created directly on the auth server, right?

Reply:
Yeah, that's only needed for in-cluster transmission between components.